### ISSUE TITLE
bug: Data doesn't update since last year

### ISSUE THREAD

**Issue by kkivan** [favna/nintendo-switch-eshop#14]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the bug

Data doesn't update since last year

### Steps To Reproduce

`await getQueriedGamesAmerica("zelda")`
Doesn't return "The Legend of Zelda™: Tears of the Kingdom"
The response of getGamesAmerica() seems the same since last year

### Expected behavior

Return "The Legend of Zelda™: Tears of the Kingdom"

**Comment by favna** [comment]
Quoting myself from the Discord server:

> The code returns whatever the Nintendo API returns. The library doesn't apply any filters.

> You are correct in that it's not there but ultimately that's up to Nintendo's API not providing the entry

> ![image](https://github.com/favna/nintendo-switch-eshop/assets/4019718/ab830963-256b-4c62-bacc-c009810d1518)

> `getQueriesGamesAmerica` does have it though

> ![image](https://github.com/favna/nintendo-switch-eshop/assets/4019718/f9e50ed7-5199-446b-bf73-dcdb7550b406)

> so I think I kinda know why

> The package specifically also fetches some common franchises in case they are omitted from the other searches (sadly the nintendo api doesnt let us just get all games outright) and that uses  `franchises:Zelda` as a filter for example.. BOTW has the franchise set, TOTK doesn't.

> **So blame Nintendo for not setting the franchise for their own game**

> yeah I investigated it some more and I really cant give you anything other than that the API just doesn't return it. It does come back from `getQueriesGamesAmerica` because that one doesn't need to do tricks to return the whole list and can just hone into 1 specific game. I'm not sure what your use-case for the nintendo-switch-api package is, but maybe you can use that method instead. For example a Discord bot is just fine using that.

TL;DR: There is nothing I can do about this. It's also a bit rough to say "last year" because afaik this is a singular case and there have been many releases between January and May.
